packages/main/src/SelectPopover.hbs

```diff
@@ -40,38 +42,57 @@
 		{{/unless}}
 
 		<ui5-list
-			selection-mode="SingleAuto"
+			mode="SingleSelectAuto"
 			separators="None"
 			@mousedown="{{_itemMousedown}}"
 			@ui5-item-click="{{_handleItemPress}}"
 		>
-			{{#each _syncedOptions}}
-				<ui5-li
-					id="{{this.id}}-li"
-					icon="{{this.icon}}"
-					?selected="{{this.selected}}"
-					?focused="{{this._focused}}"
-					tooltip="{{this.tooltip}}"
-					additional-text="{{this.additionalText}}"
-					?aria-selected="{{this.selected}}"
-					data-ui5-stable="{{this.stableDomRef}}"
-				>
-					{{this.textContent}}
-				</ui5-li>
+			{{#each options}}
+				{{#if this.isCustom}}
+					<ui5-li-custom
+						id="{{this._id}}-li"
+						tooltip="{{this.tooltip}}"
+						data-ui5-id="{{this._id}}"
+						icon="{{this.icon}}"
+						?selected="{{this.selected}}"
+						?focused="{{this.focused}}"
+						title="{{this.title}}"
+						additional-text="{{this.additionalText}}"
+						?aria-selected="{{this.selected}}"
+						data-ui5-stable="{{this.stableDomRef}}">
+						<slot name="{{this._individualSlot}}"></slot>
+					</ui5-li-custom>
+				{{else}}
+					<ui5-li
+						id="{{this._id}}-li"
+						tooltip="{{this.tooltip}}"
+						data-ui5-id="{{this._id}}"
+						icon="{{this.icon}}"
+						?selected="{{this.selected}}"
+						?focused="{{this.focused}}"
+						title="{{this.title}}"
+						additional-text="{{this.additionalText}}"
+						?aria-selected="{{this.selected}}"
```

Comment:
do we need that aria attribute? i think that it already comes from selected property of `ui5-li`. Same apply for ui5-li-custom